lib/repo-type-mappings/yocto-based-OS-image/versionist.conf.js

```diff
@@ -5,8 +5,18 @@ const fs = require('fs')
 const path = require('path')
 const semver = require('balena-semver')
 const shell = require('shelljs')
+const yaml = require('js-yaml');
+
+const isESR = (version) => {
+  return /^\d{4}.(01|04|07|10).\d*$/.test(version)
```

Comment:
```suggestion
  return /^\d{4}.(01|04|07|10).\d+$/.test(version)
```
Assuming we want one or more numbers after the last period, and prevent matching `1234.07.`

Reply:
Also I guess we don't need to escape the `.`s? I would have expected `\.` but you would have tested it.

Reply:
Totally `^\d{4}\.(01|04|07|10)\.\d+$` is more restrictive. Given that the ESR branches and versioning are automated I was too lax on invalid pattern testing.